octgnFX/Octgn/Play/Card.cs

```diff
@@ -502,39 +494,39 @@ public bool IsFiltered
             get { return _filter != null; }
         }
 
-        public Span CardText
+        public Dictionary<PropertyDef, object> CurrentCardProperties
         {
             get
             {
-                var ret = new Span();
-                ret.Inlines.Add("Card");
+                var ret = new Dictionary<PropertyDef, object>();
                 if (FaceUp && _type.Model != null)
                 {
-                    ret = new Span();
-                    ret.Inlines.Add(new Run(Name) { FontWeight = FontWeights.Bold });
                     foreach (var gameProperty in Program.GameEngine.Definition.CustomProperties)
                     {
                         if (!gameProperty.Hidden && !gameProperty.IgnoreText)
                         {
                             var cardProperty = GetProperty(gameProperty.Name, alternate: Alternate());
                             if (cardProperty != null)
                             {
-                                ret.Inlines.Add(new LineBreak());
-                                ret.Inlines.Add(new Run(gameProperty.Name + ": ") { FontWeight = FontWeights.Bold });
-                                if (gameProperty.Type == PropertyType.RichText)
-                                {
-                                    var richTextSpan = RichTextConverter.ConvertToSpan(cardProperty as RichTextPropertyValue, Program.GameEngine.Definition);
-                                    ret.Inlines.Add(richTextSpan);
-                                }
-                                else
-                                {
-                                    ret.Inlines.Add(cardProperty.ToString().Trim());
-                                }
-
+                                ret[gameProperty] = cardProperty;
                             }
                         }
                     }
                 }
+
+                return ret;
+            }
+        }
+
+        public string AccessibilityCardInfo
+        {
+            get
+            {
+                string ret = "";
+                foreach (var prop in CurrentCardProperties)
+                {
+                    ret += string.Format("\n{0}: {1}", prop.Key.Name, prop.Value.ToString());
```

Comment:
maybe I'm just paranoid but...
```suggestion
                    ret += string.Format("\n{0}: {1}", prop.Key.Name, prop.Value.ToString().Trim());
```